helm/chart-operator/values.yaml

```diff
@@ -9,6 +9,8 @@ externalDNSIP: 8.8.8.8
 helm:
   http:
     clientTimeout: "5s"
+  kubernetes:
+    waitTimeout: "10s"
```

Comment:
Increasing the wait timeout from `3s` to `10s`. Except in China where we will use `30s`.

Reply:
Can we test this value with a prometheus-operator? I think it's valid to check with the chart which takes the longest hours.

Reply:
Good call. I'll do that later on another CP. I've been testing on giraffe but its so slow its unusable. 

I don't want us to block because prometheus-operator takes too long. If this doesn't work then we may need to fix the prometheus-operator chart and try and make it faster.

Reply:
I tested with prometheus-operator on giraffe and it worked fine. But I want to test more on giraffe in the morning.